packages/fast-usdc/src/exos/liquidity-pool.js

```diff
@@ -262,15 +262,15 @@ export const prepareLiquidityPoolKit = (zone, zcf, USDC, tools) => {
       },
 
       depositHandler: {
-        /** @param {ZCFSeat} lp */
-        async handle(lp) {
+        /** @param {ZCFSeat} lpSeat */
+        async handle(lpSeat) {
```

Comment:
I lightly oppose having the type in the name. I think the existing names were clear

Reply:
Good reasons against `mint` but not to retain `Seat`. The union of our points yields: `lp` and `sharePayout`.